source/test/java/com/restfb/types/PostTest.java

```diff
@@ -294,4 +297,30 @@ public void checkV2_8_feedTargetingScheduleUntil() {
     Date ts = examplePost.getFeedTargeting().getRelevantUntilTs();
     assertEquals(1457620560000L, ts.getTime());
   }
+
+  @Test
+  public void checkV2_5_feedTargetingWithRegions() {
+    Post examplePost = createJsonMapper().toJavaObject(jsonFromClasspath("v2_5/post-with-feedtargeting-regions"), Post.class);
```

Comment:
There is something strange. I just checked an example post with graph API 2.5 and 2.8. I receive these jsons:

API 2.5
```javascript
{
	"targeting": {
		"cities": [{
			"id": 542609,
			"name": "Berlin"
		}],
		"countries": [
			"DE"
		]
	},
	"feed_targeting": {
		"cities": [{
			"id": 542609,
			"name": "Berlin"
		}],
		"countries": [
			"DE"
		]
	},
	"id": "349489628480201_838440639585095"
}
```

API 2.8
```javascript
{
	"targeting": {
		"geo_locations": {
			"countries": [
				"DE"
			],
			"cities": [{
				"id": 542609,
				"name": "Berlin"
			}]
		}
	},
	"feed_targeting": {
		"cities": [{
			"id": 542609,
			"name": "Berlin"
		}],
		"countries": [
			"DE"
		]
	},
	"id": "349489628480201_838440639585095"
}
```

As you can see, only the version 2.8 contains the `geo_locations` field in the `targeting` object. In 2.5 the field is not present.

Can you please check your json. Something is different, because your 2.5 and 2.8 JSON look very equal ...